CONTRIBUTING.md

```diff
@@ -85,6 +89,75 @@ Because we will only merge a Pull Request for which all tests pass. The followin
   See [About TypeScript demos](#about-typescript-demos).
 - The Pull Request title follows the pattern `[Component] Imperative commit message`. (See: [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/#imperative) for a great explanation)
 
+#### Checks and how to fix them
+
+If any of the checks fails click on the _Details_
+link and checkout the logs of the build to find out why it failed. The following
```

Comment:
```suggestion
link and review out the logs of the build to find out why it failed. The following
```

Reply:
Did you mean only `review` or `review out`? Never heard the latter one.